app/controllers/votes_controller.rb

```diff
@@ -1,12 +1,20 @@
 class VotesController < ApplicationController
   before_action :set_challenge, only: %i[new]
   def create
+    @vote = Vote.new(player: Player.find(params[:player]), user: current_user)
+    @vote.user = current_user
+    @vote.save
+    # redirect_to new_challenge_vote(current_user.next_votable_challenge.id)
```

Comment:
why not redirect to the next challenge?

redirect_to new_challenge_vote(current_user.next_votable_challenge)